src/api/comments.ts

```diff
@@ -0,0 +1,14 @@
+import { CommentData, Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
+
+export const addComment = (newComment: CommentData) => {
```

Comment:
```suggestion
export const addComment = (newComment: Omit<Comment, 'id'>) => {
```
You can use 'Omit' to exclude some key from type)

Reply:
Agree)
 Did that because that type (CommentData) had already been in the task)